wgsl/index.bs

```diff
@@ -1312,6 +1312,13 @@ Note: When no conversion is performed, the conversion rank is zero.
       <td>There are no automatic conversions between other types.
 </table>
 
+[=Vectors=] and [=matrix|matrices=] of [=abstract numeric types=] use the same
```

Comment:
LGTM.


I had a thought that we could also explicitly codify this in the table tool.   I can do a followup for that.